src/Components/Aspire.RabbitMQ.Client/RabbitMQClientSettings.cs

```diff
@@ -34,4 +34,16 @@ public sealed class RabbitMQClientSettings
     /// The default value is <see langword="false"/>.
     /// </value>
     public bool DisableTracing { get; set; }
+
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether auto activation is disabled or not.
+    /// </summary>
+    /// <remarks>
+    /// When auto activation is enabled, the RabbitMQ connection is established at startup time rather than on first use,
+    /// which prevents blocking threads when the connection is first requested from the DI container.
+    /// </remarks>
+    /// <value>
+    /// The default value is <see langword="true"/>.
+    /// </value>
+    public bool DisableAutoActivation { get; set; } = true;
```

Comment:
Assuming this is this way because we expect `false` later, keeping `true` so it's not breaking.

Should this be `EnableAutoActivation = false` since the gesture are to set `true` to update the default and accept a breaking change in API when we decide to use a different default?

Reply:
> Assuming this is this way because we expect false later, keeping true so it's not breaking.

Correct. The intention is that we will flip the default in the next version.

> and accept a breaking change in API when we decide to use a different default?

I was trying not to make a breaking change in the future. While this is a little awkward for one release, the naming fits with the long term intention of this setting.